starcli/layouts.py

```diff
@@ -89,3 +92,45 @@ def table_layout(repos):
 
     console = Console()
     console.print(table)
+
+def grid_layout(repos):
+    """ Displays repositories in a grid format using rich """
+
+    max_description_len = 90
+
+    panels = []
+    for repo in repos*4:
```

Comment:
Did you do the times 4 because the default number of results is too little? If so you can change the default number of results :)

Reply:
Oh whoops, nope did not mean to leave that there. Thanks!